src/header.rs

```diff
@@ -207,9 +207,9 @@ impl Default for NiftiHeader {
             quatern_y: 0.,
             quatern_z: 0.,
 
-            srow_x: [0.; 4],
-            srow_y: [0.; 4],
-            srow_z: [0.; 4],
+            srow_x: [1., 0., 0., 0.,],
+            srow_y: [0., 1., 0., 0.,],
+            srow_z: [0., 0., 1., 0.,],
```

Comment:
I asked my colleague again, just to be sure that he hasn't changed his mind in the last 6 months. In his [educated] opinion, we should set identity to sform AND qform, and set both codes to 1. Of course, the standard is not clear on everything, so setting both to the right values is a safe bet.

If you don't want to set both, then at least set `sform_code` to 1, otherwise it tells the reader to ignore the sform matrix. At least one of the `code` should be `!= 0`.